packages/shared/src/components/modals/ShareModal.tsx

```diff
@@ -103,8 +103,18 @@ export default function ShareModal({
           value={link}
           readOnly
         />
-        <p className="py-2.5 font-bold typo-callout">Share to</p>
         <SocialShare
```

Comment:
Why do we load this dynamically?
Is it not simpler to have both inside the social share directly?

Reply:
I still think these two <SocialShare> can just be inside the SocialShare component.